iothub/device/src/Convention/WritableClientPropertyCollection.cs

```diff
@@ -86,15 +86,20 @@ public bool Contains(string componentName, string propertyName)
         }
 
         /// <summary>
-        /// Gets the value of a root-level property.
+        /// Gets the value of a root-level property as a <see cref="WritableClientProperty"/>.
         /// </summary>
+        /// <remarks>
+        /// <see cref="WritableClientProperty"/> has a convenience method to help you build the writable property acknowledgement object.
+        /// To retrieve the value of the root-level writable property update request see <see cref="TryGetValue{T}(string, out T)"/>
+        /// or <see cref="WritableClientProperty.TryGetValue{T}(out T)"/>.
+        /// </remarks>
         /// <param name="propertyName">The property to get.</param>
-        /// <param name="propertyValue">When this method returns true, this contains the value of the root-level property.
+        /// <param name="writableClientProperty">When this method returns true, this contains the value of the root-level property.
```

Comment:
```suggestion
        /// <param name="writableClientProperty"> When this method returns true, this contains the value of the root-level property.
```
```suggestion
        /// <param name="writableClientProperty">When this method returns true, this contains the value of the root-level property.
```

Reply:
We don't add whitespace between the param name and the description so I'm going to leave it as-is.